tests/fixtures/noqa/noqa.py

```diff
@@ -711,3 +711,8 @@ def consecutive_yields():
         # https://github.com/wemake-services/wemake-python-styleguide/issues/1082
         break
     my_print(4)
+
+f1 = 3.0
```

Comment:
You can use just `my_print(some_var == 3.0)` 
We tend to use the simplest example in this file.